docs/core/tools/dotnet-aspnet-codegenerator.md

```diff
@@ -0,0 +1,191 @@
+---
+title: dotnet aspnet-codegenerator command
+author: rick-anderson
+ms.author: riande
+description: The dotnet aspnet-codegenerator command scaffolds ASP.NET Core projects
+ms.date: 06/15/2019
+---
+# dotnet aspnet-codegenerator
+
+`dotnet aspnet-codegenerator` - Runs the ASP.NET Core scaffolding engine.
+
+[!INCLUDE [topic-appliesto-net-core-21plus.md](../../../includes/topic-appliesto-net-core-21plus.md)]
+
+## Installing aspnet-codegenerator
+
+`aspnet-codegenerator` is a [global tool](global-tools.md) that must be installed. The following command installs the latest stable version of the `aspnet-codegenerator` tool:
+
+```console
+dotnet tool install -g aspnet-codegenerator
+```
+
+The following command updates `aspnet-codegenerator` to the latest stable version available from the installed .NET Core SDKs:
+
+```console
+dotnet tool update -g aspnet-codegenerator
+```
+
+## Synopsis
+
+```
+dotnet aspnet-codegenerator [arguments] [-p|--project] [-n|--nuget-package-dir] [-c|--configuration] [-tfm|--target-framework] [-b|--build-base-path] [--no-build] 
+dotnet aspnet-codegenerator [-h|--help]
+```
+
+## Description
+
+The `dotnet aspnet-codegenerator ` global command runs the ASP.NET Core code generator and scaffolding engine.
+
+## Arguments
+
+`generator`
+
+The code generator to run. The following generators are available:
+
+| Generator | Operation |
+| ----------------- | ------------ | 
+| area      | Generates an [Area](/aspnet/core/mvc/controllers/areas) |
```

Comment:
Consider making the wording in the operation column consistent for all the scaffolders.